src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/LanguageServiceHost.cs

```diff
@@ -128,34 +128,14 @@ Task ILanguageServiceHost.InitializeAsync(CancellationToken cancellationToken)
             }
         }
 
-        private JoinableTask<T> ExecuteWithinLockAsync<T>(Func<Task<T>> task)
+        private Task<T> ExecuteWithinLockAsync<T>(Func<Task<T>> task)
```

Comment:
Fixed these, appreciate you telling the exact lines - I'm not getting the build errors in the parent commit this applied to.